jiant/preprocess.py

```diff
@@ -650,6 +652,14 @@ def add_task_label_vocab(vocab, task):
     This can then be accessed when generating Instances, either via a custom
     Indexer or by invoking the namespace when creating a LabelField.
     """
+    if isinstance(task, MLMTask):
+        tokenizer =  RobertaTokenizer.from_pretrained("roberta-large")
```

Comment:
can you try to generalize this for other input modules? you should be able to get corresponding tokenizer by calling "get_tokenizer"

Reply:
Ah yes will do